scalyr_agent/builtin_monitors/kubernetes_events_monitor.py

```diff
@@ -350,6 +351,7 @@ def _initialize(self):
                 "ReplicaSet",
                 "ReplicationController",
                 "StatefulSet",
+                "Endpoint",
```

Comment:
Actually since this type is new, please also add a corresponding changelog entry.